packages/developer-portal/src/components/pages/apps/edit/form-schema/validation-schema.ts

```diff
@@ -121,8 +121,8 @@ export const appEditValidationSchema = object().shape({
       otherwise: string()
         .trim()
         .test({
-          name: 'isValidSummary',
-          message: errorMessages.BETWEEN_MIN_MAX_CHARACTER_LENGTH(50, 150),
+          name: 'isValidDescription',
+          message: errorMessages.BETWEEN_MIN_MAX_CHARACTER_LENGTH(150, 1500),
```

Comment:
Does this mean if I had an app with a description lower than 150 char, I will be forced into updating it if I need to edit my app?

Reply:
It's always been 150 chars, I got the validation wrong!